src/value.h

```diff
@@ -4,7 +4,9 @@
 #include "llvm/ADT/APFloat.h"
 #include "mlir/Dialect/Linalg/IR/LinalgOps.h"
 #include <string>
+#include <optional>
 #include <vector>
+#include <concepts>
```

Comment:
Is this needed?

Reply:
Oops... forgot to remove this while testing some changes